Makefile

```diff
@@ -1,8 +1,12 @@
 VERSION := v0.1.0
 
+# Windows OS Version
+OSVERSION ?= 1809
```

Comment:
see my previous comment about multiple OS versions since it needs to match the host